met-api/src/met_api/models/comment.py

```diff
@@ -81,15 +77,15 @@ def get_comments_by_survey_id_paginated(cls, survey_id, pagination_options: Pagi
     def get_accepted_comments_by_survey_id_where_engagement_closed_paginated(
             cls, survey_id, pagination_options: PaginationOptions):
         """Get comments for closed engagements."""
-        now = datetime.now()
         query = db.session.query(Comment)\
+            .join(Submission)\
             .join(CommentStatus)\
```

Comment:
Maybe it's only me, I feared I didn't understand how CommentStatus was joined here, is it possible to add the On part?